File: Data Structures & Algorithms/online-stock-span/submission-0.py
class StockSpanner:

    def __init__(self):
        self.stack = []

    def next(self, price: int) -> int:
        self.stack.append(price)
        res = 0
        maxw = -1
        for num in self.stack:
            if num <= price:
                res += 1
            else:
                res = max(res, maxw)
                res = 0
        return res


# Your StockSpanner object will be instantiated and called as such:
# obj = StockSpanner()
# param_1 = obj.next(price)

File: Data Structures & Algorithms/design-hashset/submission-0.cpp
class MyHashSet {
public:
    unordered_set<int> hashset;

    MyHashSet() {}
    
    void add(int key) {
        hashset.insert(key);
    }
    
    void remove(int key) {
        if (hashset.contains(key)) hashset.erase(key);
    }
    
    bool contains(int key) {
        return hashset.contains(key);
    }
};

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet* obj = new MyHashSet();
 * obj->add(key);
 * obj->remove(key);
 * bool param_3 = obj->contains(key);
 */